input/pagecontent/reference.md

```diff
@@ -1842,6 +1836,42 @@ Any FSH syntax errors that arise as a result of the value substitution are handl
   // more rules
   ```
 
+##### Inserting Rule Sets at Elements
```

Comment:
"Inserting Rule Sets _at_ Elements" feels just a little weird to me.  I can't decide if it's right or not.  Other contenders:
* Inserting Rule Sets _on_ Elements
* Inserting Rule Sets _in_ Elements
* Inserting Rule Sets at _Paths_
* ...?

@markkramerus -- any thoughts on the wording?

Reply:
Yes, we used "context" for indented rules and this is very similar. Made the correction.